WinUIGallery/Samples/ControlPages/Design/IconographyPage.xaml

```diff
@@ -33,14 +33,27 @@
             <Setter Property="BorderBrush" Value="Transparent" />
             <Setter Property="BorderThickness" Value="1" />
             <Setter Property="MinHeight" Value="32" />
-            <Setter Property="Padding" Value="24" />
             <Setter Property="CornerRadius" Value="0" />
             <Setter Property="Background" Value="Transparent" />
             <Setter Property="SampleType" Value="Inline" />
             <Setter Property="VerticalContentAlignment" Value="Center" />
             <Setter Property="Margin" Value="0,0,0,8" />
         </Style>
 
+        <Style
```

Comment:
I would prefer not creating a style just to set a single property, this seems a bit overkill.

Reply:
I understand your point.🫡